modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -95,6 +95,10 @@ class TopLevelNodes extends React.Component {
             this.props.model.viewState.packageDefExpanded = false;
         }
         if (this.packageDefValue) {
+            // If there is a semi-colon at the end of the package name, remove it
+            if (this.packageDefValue.slice(-1) === ';') {
```

Comment:
Or use this.packageDefValue = _.trimEnd(this.packageDefValue, ';') , so  you can get rid of this whole block.

Reply:
Fixed with commit 05f4d18